### ISSUE TITLE
Battery App opens in full setting on every restart and needs to be closed. Options to not open it do not work. Colour selection for full charge does not remain selected.

### ISSUE THREAD

**Issue by Velkyhores** [soleon/Percentage#19]

**Comment by soleon** [comment]
This is duplicated with issue #4 which has been resolved. Closing this one.

Please try uninstalling and reinstalling the latest version from Microsoft Store to resolve this issue.

**Comment by Velkyhores** [comment]
Thanks will try but I have done this already a few days ago without anly luck.


Regards,

Leslie Klein, Ph.D., P.Eng<http://P.Eng>.
President and CEO
C-COM Satellite Systems Inc.
2574 Sheffield Road
Ottawa, Canada, K1B 3V7

Direct Line:613-986-4110
email: ***@***.******@***.***>
web site: www.c-comsat.com<http://www.c-comsat.com>

Trading Symbol: CMI on
TSX Venture (Canada) and
CYSNF on OTCQB (USA)

________________________________

**Comment by Velkyhores** [comment]
Hi Long,
I have uninstalled and reinstalled as you suggested, but the start up process still brings up the entire setting page even though I set the option to hide. See my setting:
***@***.***

**Comment by soleon** [comment]
@Velkyhores can you try deleting the following file  then reset all settings to default in the app see if this helps?
```
C:\Users\<user name>\AppData\Local\Percentage.App\Percentage.App_Url_<unique ID>\1.0.0.0\user.config
```

**Comment by Velkyhores** [comment]
Hi Long,
I do not see Percentage.App in that folder..

**Comment by soleon** [comment]
@Velkyhores can you follow [my comment](https://github.com/soleon/Percentage/issues/22#issuecomment-2556131886) in issue #22 see if it can help you solving this issue please.

**Comment by Velkyhores** [comment]
Will try. Thanks for the assistance. Having problems locating the installation folder for the app on my drive.


Regards,

Leslie Klein, Ph.D., P.Eng<http://P.Eng>.
President and CEO
C-COM Satellite Systems Inc.
2574 Sheffield Road
Ottawa, Canada, K1B 3V7

Direct Line:613-986-4110
email: ***@***.******@***.***>
web site: www.c-comsat.com<http://www.c-comsat.com>

Trading Symbol: CMI on
TSX Venture (Canada) and
CYSNF on OTCQB (USA)

________________________________

**Comment by Velkyhores** [comment]
Hi Long,
Folder does not exist in suggested location. Issue is now only with startup which I can close. Everything else is OK and colour selection remains saved.

**Comment by Velkyhores** [comment]
Hi Long,
The app started to work out of the blue. No issues. Nothing has changed and I was unable to make any changes. Total mistery.
Thanks for your effort.

From: Leslie Klein
Sent: December 19, 2024 8:13 AM
To: soleon/Percentage ***@***.***>
Subject: RE: [soleon/Percentage] Battery App opens in full setting on every restart and needs to be closed. Options to not open it do not work. Colour selection for full charge does not remain selected. (Issue #19)

Hi Long,
I have uninstalled and reinstalled as you suggested, but the start up process still brings up the entire setting page even though I set the option to hide. See my setting:
***@***.***
